cegs_portal/search/templates/search/v1/experiment_list.html

```diff
@@ -1,39 +1,140 @@
-{% extends "base.html" %}
-{% load static i18n %}
+{% extends "base.html" %} {% load static i18n %} {% block css %} {{ block.super }}
+<style>
+    .faded-line {
+        border-top: 0.5px solid rgb(133, 139, 146);
+        margin-top: 10px;
+    }
+
+    .content-container {
+        background: #ffffff;
+        box-shadow: 0px 2px 5px 4px rgba(0, 0, 0, 0.1);
+        display: flex;
+        align-items: center;
+        padding: 20px 40px;
+        margin-bottom: 20px;
+        border-radius: 10px;
+    }
+
+    .content-container-link:hover .content-container {
+        transform: scale(1.02);
+        box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
+    }
+
+    .content-container-link {
+        text-decoration: none;
+        color: inherit;
+    }
+
+    .icon-container {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 3rem;
+        height: 3rem;
+        margin-right: 1rem;
+    }
+
+    .accession-id {
+        color: #3490dc;
+        font-weight: bold;
+        text-decoration: none;
+    }
+
+    .accession-id {
```

Comment:
You should combine accession-id classes into one class.

Reply:
Agreed. Classes have been combined.